sdk/communication/communication-phone-numbers/src/models.ts

```diff
@@ -81,7 +88,12 @@ export interface ListSipRoutesOptions extends OperationOptions {}
 /**
  * Additional options that can be passed to list SIP trunks.
  */
-export interface ListSipTrunksOptions extends OperationOptions {}
+export interface ListSipTrunksOptions extends SipRoutingGetOptionalParams {}
+
+/**
+ * Additional options that can be passed to get SIP trunks.
+ */
+export interface GetSipTrunksOptions extends SipRoutingGetOptionalParams {}
```

Comment:
Feedback from review:
Same comment as for "ListSipTrunksOptions", so something like this:

export interface GetSipTrunkOptions extends OperationOptions {
    includeHealth?: boolean;
}

Also, map the boolean property appropriately on the ExpandEnum behind the scenes (don't expose ExpandEnum anymore).

Reply:
Updated.